definitions/npm/@testing-library/react_v9.x.x/flow_v0.104.x-/react_v9.x.x.js

```diff
@@ -115,16 +115,8 @@ declare module '@testing-library/react' {
     eventProperties?: TInit
   ) => boolean;
 
-  declare type RenderResult<Queries = GetsAndQueries> = {|
-    container: HTMLDivElement,
-    unmount: () => void,
-    baseElement: HTMLElement,
-    asFragment: () => DocumentFragment,
-    debug: (baseElement?: HTMLElement) => void,
-    rerender: (ui: React$Element<*>) => void,
-  |} & Queries;
-  declare type RenderResultWithCustomQueries<CustomQueries> = {
-    ...CustomQueries,
+  declare type RenderResult<Queries = GetsAndQueries> = {
```

Comment:
Now this type is no longer exact (as I can't think of a way to specify that `Queries` could be exact). Maybe this is still better than having two different types for the render result as we did after #3705.

Reply:
Not 100% sure that spreading inexact objects is a good thing, but definitely better than an intersection on an exact one :) Plus less type duplication. Let's try.